addon/i3dio/ui/object.py

```diff
@@ -131,6 +139,91 @@ class I3DNodeObjectAttributes(bpy.types.PropertyGroup):
         default=i3d_map['trigger']['default']
     )
 
+    restitution: FloatProperty(
+        name="Restitution",
+        description="Bounciness of the surface",
+        default=i3d_map['restitution']['default'],
+        min=0,
+        max=1
+    )
+
+    static_friction: FloatProperty(
+        name="Static Friction",
+        description="The force that resists motion between two non-moving surfaces",
+        default=i3d_map['static_friction']['default'],
+        min=0,
+        max=1
+    )
+
+    dynamic_friction: FloatProperty(
+        name="Dynamic Friction",
+        description="The force that resists motion between two moving surfaces",
+        default=i3d_map['dynamic_friction']['default'],
+        min=0,
+        max=1
+    )
+
+    linear_damping: FloatProperty(
+        name="Linear Damping",
+        description="Defines the slowdown factor for linear movement, affecting speed",
+        default=i3d_map['linear_damping']['default'],
+        min=0,
+        max=1
+    )
+
+    angular_damping: FloatProperty(
+        name="Angular Damping",
+        description="Defines the slowdown factor for angular movement, affecting spin",
+        default=i3d_map['angular_damping']['default'],
+        min=0,
+        max=1
+    )
+
+    density: FloatProperty(
+        name="Density",
+        description="Used with the shape of the object to calculate mass."
+                    "The higher the number, the heavier the object",
+        default=i3d_map['density']['default'],
+        min=0,
+        max=20
+    )
+
+    split_type: EnumProperty(
+        name="Split Type",
+        description="Split type determines what type of tree it is."
+                    "For custom tree type use a number over 19",
+        items=[
+            ('0', "None", "None"),
+            ('1', "Spruce", "Spruce"),
+            ('2', "Pine", "Pine"),
+            ('3', "Larch", "Larch"),
+            ('4', "Birch", "Birch"),
+            ('5', "Beech", "Beech"),
+            ('6', "Maple", "Maple"),
+            ('7', "Oak", "Oak"),
+            ('8', "Ash", "Ash"),
+            ('9', "Locust", "Locust"),
+            ('10', "Mahogany", "Mahogany"),
+            ('11', "Poplar", "Poplar"),
+            ('12', "AmericanElm", "American Elm"),
+            ('13', "Cypress", "Cypress"),
+            ('14', "DownyServiceberry", "Downy Serviceberry"),

```

Comment:
The tree types is numbers. Could probably change the names to use space instead of camel case.

Reply:
Ahhh, in the i3d I suppose?

I think the names at least should be changed to look like the descriptions you did

Reply:
Yeah in the i3d 😄

Yup agree with you 🙈

Reply:
Once you fix that I will approve the PR ;-)

Reply:
One more question. You write "For custom tree type use a number over 19". But isn't it an enum property aka you get to select between those values only?

Reply:
Ah crap I forgot about that... Perhaps what I could do is just add a integer prop where the user can input what ever value they want and the enum can be like preset values from the game and it will write to the integer prop when the user select something from the enum

Reply:
You could have an integer as the actual property that gets exported and then a enum with a "Custom" option. You don't render the integer unless the enum is set to "Custom" and when the enum is set to anything it should update the integer behind the scenes. Then you should probably make the enum not export somehow. You could abuse the "depends" dependency in the i3d_map to depend on something that will never be true (you can look in the ui/light.py for how "depends" is specified)